mobile-modal.js

```diff
@@ -0,0 +1,74 @@
+const EXPERIENCE_GRID_MOBILE = document.querySelector('.experience__grid');
+const GRID_MOBILE_PROJECT = [
+  {
+    title: 'Profesional Art Printing Data',
+    itemStack: ['Bootstrap', 'HTML', 'Ruby'],
+    img: 'Assets/Snapshoot Portfolio.svg',
+    pText: "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    button: 'See project',
+  },
+  {
+    title: '2Profesional Art Printing Data',
+    itemStack: ['Bootstrap', 'HTML', 'Ruby'],
+    img: 'Assets/Snapshoot Portfolio.svg',
+    pText:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    button: 'See project',
+  },
+  {
+    title: '3Profesional Art Printing Data',
+    itemStack: ['Bootstrap', 'HTML', 'Ruby'],
+    img: 'Assets/Snapshoot Portfolio.svg',
+    pText:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    button: 'See project',
+  },
+  {
+    title: '4Profesional Art Printing Data',
+    itemStack: ['Bootstrap', 'HTML', 'Ruby'],
+    img: 'Assets/Snapshoot Portfolio.svg',
+    pText:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    button: 'See project',
+  },
+  {
+    title: '5Profesional Art Printing Data',
+    itemStack: ['Bootstrap', 'HTML', 'Ruby'],
+    img: 'Assets/Snapshoot Portfolio.svg',
+    pText:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    button: 'See project',
+  },
+  {
+    title: 'Profesional Art Printing Data',
+    itemStack: ['Bootstrap', 'HTML', 'Ruby'],
+    img: 'Assets/Snapshoot Portfolio.svg',
+    pText:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    button: 'See project',
+  },
+];
+
+const cardMobile = (projectInfo) => {
+  const {
+    title, pText, itemStack, button,
+  } = projectInfo;
+
+  return `<div class="grid__item">
+    <h4>${title}</h4>
+    <p>${pText}</p>
+    <ul class="grid__item-stack">
+      ${itemStack
+    .map((item) => `<li class="grid__badges">${item}</li>`)
+    .join('')}
+    </ul>
+    <button class="grid__item-button experience__button 345345345">${button}</button>
+  </div>`;
+};
+
+const gridMobile = (gridProjects) => {
+  const markup = gridProjects.reduce((acc, item) => acc + cardMobile(item), '');
+  return markup;
+};
```

Comment:
You have had a good implementation of the modal however, there are a couple of problems as stated below:

- On the mobile, the modal is not being triggered on the work cards except for the very first
- On the desktop, the positioning of the modal on the screen is not proper, attached below is a screenshot of how it looks
- I think you do not need a separate modal for the mobile also, you can use the same for the desktop and all you need to do is make it responsive and that's all

I recommend taking the above-mentioned suggestion as they would improve the look of your page.

here is the desktop
![Screenshot](https://github.com/sinruidotec/Desktop-Version-Portfolio/assets/84930355/0498a493-deed-4db0-9fd8-a958870d2fd9)

the position is the same as the one that managed to appear on the mobile



[OPTIONAL] I recommend that you don't hide the button on hover, it makes it behave unexpectedly when the modal is attempted to be interacted with